docs/application/dotnet/guides/applications/nui-widget-app.md

```diff
@@ -0,0 +1,243 @@
+
+## Application and Viewer
+
+[`NUIWidgetApplication`](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.NUIWidgetApplication.html) makes diverse class instances whenever [`WidgetView`](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.WidgetView.html) requests for a widget instance.
+
+The widget instance has its own life-cycle similar to that of the widget application. However, the widget instance is an object created by the widget viewer application. Many widget instances can be running on a widget application process.
+
+> [!NOTE]
+> The case to use many widget instances in one widget application is known as multi-instance. In some devices, the multi-instance may not be supported. If a device does not support multi-instance, an error message is displayed.
+
+[`WidgetView`](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.WidgetView.html) shows the contents drawn by [`Widget`](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.Widget.html) on the screen.
+
+To summarize, you [create a NUI WidgetApplication](#create-a-nui-widget-application), [create a NUI WidgetView](#create-a-nui-widget-view), and use Widget to view layout on the screen.
+
+## Instance States and Events
+
+The `Widget` class provides interface for creating custom widget.
+The following table lists the main functions to manage the widget instance:
+
+| Function           | Description                                                                                   |
+|--------------------|-----------------------------------------------------------------------------------------------|
+| `OnCreate()`       | This function is called after the widget instance is created.                                 |
+| `OnTerminate()`    | This function is called after the widget instance is terminated.                              |
+| `OnPause()`        | This function is called when the widget is invisible.                                         |
+| `OnResume()`       | This function is called when the widget is visible.                                           |
+| `OnResize()`       | This function is called before the widget size is changed.                                    |
+| `OnUpdate()`       | This function is called when an event for updating the widget is received.                    |
+| `SetContentInfo()` | This function sends data about current state of widget instance to a widget viewer application. If you want to save the current state of widget before deleting it, you can save it to the current status using `SetContentInfo()`.|
+
+Most of functions are used for managing widget instance life-cycle.
+
+## Prerequisites
+To enable your application to use the NUI widget functionality:
+
+To use the methods and properties of [Tizen.NUI](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.html) namespace, include it in your application:
```

Comment:
I agree. 
@tscholb,
Please update.

Reply:
i think 204~212 and 64~66 is different case.
64~66 is for NUI WidgetApplication and 204~212 is for NUI WidgetVIewer.
in my  opinion,  it need to divide to avoid confuse. how about your think?
Please let me know if there is a better way.

Reply:
@tscholb, as per my understanding sicne the prerequisite section si common for both in that case we can go with @mijin85cho suggestion. Please correct me if I am wrong.

Reply:
I suggested to move this context to prerequisites section as I thought adding privileges is also a prerequisite to use the NUI WidgetViewer.
As @surbhig12 replied, the prerequisites section is for overall cases.

If you want to emphasize that the privileges are for the NUI WidgetViewer,
I think we can change the sentence a little bit.

...

2. To use the \[NUI WidgetVIewer](API link), the application has to request permission by adding the following privileges to the \`tizen-manifest.xml\` file:

    ```XML
      <privileges>
        <privilege>http://tizen.org/privilege/widget.viewer</privilege>
        <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
      </privileges>
    ```
...

Would you please let me know your opinion about this suggestion?

Reply:
okay i understand. i will updated it